lib/benchee/conversion.ex

```diff
@@ -18,37 +18,39 @@ defmodule Benchee.Conversion do
   ## Examples
 
       iex> statistics = %Benchee.Statistics{average: 1_000_000.0, ips: 1000.0}
-      iex> scenario = %Benchee.Scenario{
+      ...> 
```

Comment:
Every `iex>` prompt that is not the first line of the test, will get changed to `...>`. This is not strictly necessary, but it's recommended by the [official docs](https://hexdocs.pm/ex_unit/ExUnit.DocTest.html#module-syntax):

> Multiline expressions can be used by prefixing subsequent lines with either ...> (recommended) or iex>.

Reply:
sounds great, thank you!